go/ct/spc/specification.go

```diff
@@ -1378,6 +1377,66 @@ func getAllRules() []Rule {
 		effect: FailEffect().Apply,
 	})...)
 
+	// --- CREATE ---
+	rules = append(rules, rulesFor(instruction{
+		op:        CREATE,
+		staticGas: 32000,
+		pops:      3,
+		pushes:    1,
+		conditions: []Condition{
+			Eq(ReadOnly(), false),
```

Comment:
The completeness check should be able to detect that there is no rule for a `CREATE` call in a Read-Only context. We can leave this as a follow-up if you like, to check the ability of the completeness check to find these problems.

Reply:
I added rules for Read-Only mode similar to the ones for SELFDESTRUCT, we can always comment out rules to evaluate the completeness check ability.